### DISCUSSION TITLE
More info in Locations view

### DISCUSSION THREAD

**Discussion by SpitFire-666** [breezy-weather/breezy-weather#1400] [Ideas]
I'm trying to move off the Weatherzone app, which has a Locations view like below. It lets me see the weather and current temp of my friends/family/holiday/home:

![image](https://github.com/user-attachments/assets/7393fdd7-8819-4794-bb92-268c08cf4c05)

Breezy Weather only shows an icon (no temp). Bonus: the local time at each location would be awesome too!

![image](https://github.com/user-attachments/assets/2db02655-c3c9-43df-8f82-0bc422b32694)

Cheers

**Comment by papjul** [comment]
#304 for localtime.

As for icon, be careful: in Breezy Weather, it's the icon for the daytime period, not the icon for current. If I add the temperature, it will be the same logic (daytime max temperature). I think there is an automatic switch between daytime and nighttime period depending on the localtime of the location, but I need to check ;)

**Reply by SpitFire-666** [reply]
Cool. FYI I'm actually wanting to see current temp in this view (like weatherzone), not max temp.

**Reply by papjul** [reply]
It's not planned, because the "current" temperature for secondary locations may be from hours ago and outdated, because as they are secondary, we avoid refreshing them as often for efficiency purposes ;)

**Reply by papjul** [reply]
We have a new design for location list that will be released as part of v5.4.2.
Current temperature will be added later, as part of #45.
I will close this discussion and let you subscribe to issue #45 to be notified when it is implemented ;)
